pkgs/applications/audio/kid3/default.nix

```diff
@@ -32,28 +32,28 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "mirror://sourceforge/project/kid3/kid3/${version}/${pname}-${version}.tar.gz";
```

Comment:
All the software under the KDE umbrella can be downloaded from here: https://download.kde.org/stable/